src/index.html

```diff
@@ -1,12 +1,87 @@
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="navbarSupportedContent" aria-controls="navbarSuportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <ion-icon name="menu-outline"></ion-icon>
+    </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav ml-auto">
+                        <li class="nav-item active">
+                            <button id="/" class="btn">Home</button>
+                            <button id="/login" class="btn">Login</button>
+                            <button id="/face" class="btn">Log in with FB</button>
+                            <button id="/about" class="btn">Postea aquí</button>
+                            <button id="/goo" class="btn">Login with Google</button>
+                        </li>
+                    </ul>
+            </nav>
+            </div>
+    </header>
+    <div id="root">
+    </div>
+
+
+    <script type="module" src="router.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/4.8.1/firebase.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.14.0/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.14.0/firebase-analytics.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.14.0/firebase.js"></script>
+
+
+    <script>
+        // Your web app's Firebase configuration
+        var firebaseConfig = {
+            apiKey: "AIzaSyDz4Hbhdbb_EAGniVZ9sZ60ZN2wg_WvWjE",
+            authDomain: "grislab-9796e.firebaseapp.com",
+            databaseURL: "https://grislab-9796e.firebaseio.com",
+            projectId: "grislab-9796e",
+            storageBucket: "grislab-9796e.appspot.com",
+            messagingSenderId: "300427088655",
+            appId: "1:300427088655:web:2263b53b6a3b79b268a47a",
+            measurementId: "G-ZR5X5SBQ10"
+        };
+        // Initialize Firebase
+        firebase.initializeApp(firebaseConfig);
+        firebase.analytics();
+    </script>
+    </div>
+    </div>
+
+    <!--<script src="js/jquery-3.2.1.slim.min.js"></script>
+<script src="js/popper.min.js"></script>
+<script src="js/bootstrap.min.js"></script>-->
+    <script src="https://unpkg.com/ionicons@5.0.0/dist/ionicons.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
 </body>
+
+</body>
```

Comment:
¿whaaat???????? 2 cierres de body ?? why??? 
esto esta muy mal
```suggestion


</body>
```

Reply:
Borren una etiqueta de cierre de body